### DISCUSSION TITLE
Custom Table

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#9472] [CLI - PWA mode]
Hello Friends 
I have a custom table i wanted to add a prefix sign ">" in the row but this will not be apply to the first and last row. i have try a solution but its applying everywhere. Please help me to find another solution 

solution i tried : i just add the **>** sign in the div
       **<div> > {{ props.row.ThresholdStart }}</div>**

![image](https://user-images.githubusercontent.com/84211129/120064018-440ac900-c07b-11eb-81b0-ea9f94853038.png)
it should look like this 

Thanks

**Comment by ontwikkelfabriek** [comment]
@akashramtohul4 

> it should look like this

Like what? For me it's not clear what the problem is and what you want to display in the table. Could you make a codepen and demonstrate your problem.
